### PR TITLE
Cleanup persistence docs, #24717

### PR DESCRIPTION
**Pull request by patriknw**
Refs #24717

### CODE REVIEW

File: akka-docs/src/main/paradox/persistence-plugins.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+```
+akka.persistence.journal.plugin = ""
+akka.persistence.snapshot-store.plugin = ""
+```
+
+However, these entries are provided as empty "", and require explicit user configuration via override in the user `application.conf`.
+For an example of a journal plugin which writes messages to LevelDB see [Local LevelDB journal](#local-leveldb-journal).
+For an example of a snapshot store plugin which writes snapshots as individual files to the local filesystem see [Local snapshot store](#local-snapshot-store).
+
```
Comment (helena) [inline akka-docs/src/main/paradox/persistence-plugins.md]: Do we want to bullet point these two? Just a thought.

**Review by chbatey — APPROVED** [review]
Still draft but all the changes look good

**Review by raboof — APPROVED** [review]
Good to split up into more pages.

Even though still a draft these all look like great improvements. Perhaps merge (after taking Chris' suggestion) and follow up in a new PR for any more pending improvements?

File: akka-docs/src/main/paradox/typed/persistence.md [hunk 2]
```diff
@@ -482,3 +584,25 @@ processed.
 
 It's allowed to stash messages while unstashing. Those newly added commands will not be processed by the
 `unstashAll` effect that was in progress and have to be unstashed by another `unstashAll`.
+
+## Scaling out
+
+In a use case where the number of persistent actors needed are higher than what would fit in the memory of one node or
```
[the remark this answers is not in the document]
Comment (raboof) [reply akka-docs/src/main/paradox/typed/persistence.md]:
```suggestion
In a use case where the number of persistent actors needed is higher than what would fit in the memory of one node or
```

**Comment by patriknw** [comment]
Added the small review fixes. Sorry I wasn't clear. It was ready, but still draft because it was on top of another PR that was to be merged first.

Will merge after validation.

File: akka-docs/src/main/paradox/persistence-plugins.md (hunk 1 above)
Comment (patriknw) [reply akka-docs/src/main/paradox/persistence-plugins.md]: changed to that

**Pull request merged**